### PR TITLE
Telemetry docs

### PR DESCRIPTION
**Pull request by ankithkonda**
new telemetry work is about to be merged in see [Private Link](https://github.com/OctopusDeploy/OctopusDeploy/pull/10665)

This PR pulls telemetry-related documentation to its own page, but is still under the same section.

### CODE REVIEW

File: docs/security/outbound-requests/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| Structured Configuration Variable usage across projects including how many steps have Structured Configuration Variables enabled and what file extensions are being used | 2020.4.0 |
+| OS Architecture of Deployment Target Tentacles | 2021.1.0 |
+| The number of Projects that are version controlled | 2021.2.0 |
+
+The installation ID is a GUID that we generate when Octopus is installed. This GUID is simply a way for us to get a rough idea of the number of installations there are in the wild, and which versions people are using, so we can make decisions about backwards compatibility support.
+
+Together, this information helps us when making decisions about the product. For example, we expected users to only have a handful of machines, but the statistics tell us that some customers have over 900; we now take that into account when designing the user experience.
+
+Be assured that names, descriptions, URI's and so on are _never_ included. You can download a preview of the data that will be sent by clicking on the "Download Preview" button on the **{{Configuration,Telemetry}}** page
```
Comment (harrisonmeister) [inline docs/security/outbound-requests/telemetry.md:55]:
```suggestion
Be assured that names, descriptions, URI's, and so on are _never_ included. You can download a preview of the data that will be sent by clicking on the **Download Preview** button on the **{{Configuration,Telemetry}}** page
```

File: docs/security/outbound-requests/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Telemetry
+description: Telemetry from Octopus Server and what is included
+position: 25
+---
+
+Telemetry reporting is enabled by default. We use the data we receive to inform our decisions and to help us make a better product. You can opt-out of sending telemetry by navigating to **{{Configuration,Telemetry}}** in Octopus.
+
+When the "Telemetry Reporting" option is enabled, Octopus will make a HTTPS request, this request includes:
```
Comment (harrisonmeister) [inline docs/security/outbound-requests/telemetry.md:9]:
```suggestion
When the **Telemetry Reporting** option is enabled, Octopus will make an HTTPS request, this request includes:
```

File: docs/security/outbound-requests/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- The current Octopus Deploy version number that you are running.
+- A unique installation ID (read more below).
+- Telemetry data payload
+
+When the "Telemetry Reporting" option is enabled, we'll send some specific aggregate criteria along with the request. This has evolved a bit over time, so it depends on the version you are running:
```
Comment (harrisonmeister) [inline docs/security/outbound-requests/telemetry.md:15]: The wording `When the "Telemetry Reporting" option is enabled` feels repetitive given its mentioned verbatim just before the preceding bullet points.

```suggestion
In addition, we'll send some specific aggregate criteria along with the request. This has evolved a bit over time, so it depends on the version you are running:
```

File: docs/security/outbound-requests/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+2. Downloading packages from the [NuGet feeds](/docs/packaging-applications/package-repositories/index.md) that you configure.
+3. Windows Azure traffic (only when deploying to an Azure deployment target).
+4. Checking for updates (if enabled).
+5. Checking for updated [community contribute step templates](/docs/projects/community-step-templates.md)  (if enabled).
+
+NOTE: Our community contributed step template integration queries `library.octopus.com` for updates.
+
+## What information is included when Octopus checks for updates? {#Outboundrequests-WhatinformationisincludedwhenOctopuschecksforupdates?}
+
+By default, Octopus will periodically check for new releases and send usage statistics to help us understand how the product is used and how we can improve it. You can opt-out of sending usage statistics or also opt-out of checking for updates by navigating to **{{Configuration,Settings,Updates & Usage Telemetry}}** in Octopus.
```
Comment (harrisonmeister) [inline docs/security/outbound-requests/index.md:34]: Question @ankithkonda - does `**{{Configuration,Settings,Updates & Usage Telemetry}}**` need updating to `**{{Configuration,Telemetry}}**` ?

Or is it just the `Telemetry` part of it no longer relevant? Maybe it should be `**{{Configuration,Settings,Updates}}** ?

**Review by harrisonmeister — COMMENTED** [review]
Some minor suggestions and one question on the opt-out of updates menu breadcrumb. Apart from that :+1:

**Comment by harrisonmeister** [comment]
@ankithkonda - I also removed the last entry in the redirects file as it was causing an infinite redirect loop on the dev site (it wasn't needed as the docs go to the index page when you move the file to a "folder" :smile:

File: docs/security/outbound-requests/index.md (hunk 4 above)
Comment (ankithkonda) [reply docs/security/outbound-requests/index.md:34]: Ahh good catch, yeah the wording around this needs to be updated, telemetry will no longer be part of check for updates, I'll fix that up 👍

**Comment by ankithkonda** [comment]
> @ankithkonda - I also removed the last entry in the redirects file as it was causing an infinite redirect loop on the dev site. It wasn't needed as the docs go to the index page when you move the file to a "folder" 😄

Ahh haha woops! thank you :)

**Review by harrisonmeister — APPROVED** [review]
This looks good to me. I'll hold off from merging it, feel free to do it if you're happy, or just let me know if it's good to go and I can do it :+1:

**Comment by ankithkonda** [comment]
Thanks @harrisonmeister :) I accidently removed the point on how to opt-out of updates, added that back in now. So this should be all good to go. I'll merge it in

**Pull request merged**